test/e2e/fips.go

```diff
@@ -19,6 +19,8 @@ const (
 )
 
 var _ = Describe("Validate FIPS Mode", func() {
+	BeforeEach(skipIfNotInDevelopmentEnv)
```

Comment:
I'm not sure that I understnad. Our e2e sub doesn't not have the flag enabled? Should we look into enabling it?